doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb

```diff
@@ -493,180 +539,374 @@
-    "theo_rdf = 1 + 1 / r_star**2 * (np.exp(-(a * r_star + b)) * np.sin(c * r_star + d) + np.exp(-(g * r_star + h)) * np.cos(k * r_star + l))\n",
-    "theo_rdf[np.nonzero(r_star <= r_star_cutoff)] = s * np.exp(-(m * r_star + n)**4)[np.nonzero(r_star <= r_star_cutoff)]\n",
-    "# plot\n",
-    "fig = plt.figure(num=None, figsize=(10, 6), dpi=80, facecolor='w', edgecolor='k')\n",
-    "fig.set_tight_layout(False)\n",
-    "plt.plot(r_star, theo_rdf, '-', color=\"blue\", linewidth=2, alpha=1, label=r'theoretical')\n",
-    "plt.plot(r, rdf, '-', color=\"#A60628\", linewidth=2, alpha=1, label='simulation')\n",
-    "plt.xlabel('r $[\\sigma]$', fontsize='xx-large')\n",
-    "plt.ylabel('$g(r)$', fontsize='xx-large')\n",
-    "plt.legend(loc='upper right', fontsize='x-large')\n",
-    "plt.show()"
+    "\n",
+    "# plot fitted expression (=theoretical curve)\n",
+    "theo_rdf_cutoff = 1.02  # slightly more than 1 to smooth out the discontinuity in the range [1.0, 1.02]\n",
+    "\n",
+    "theo_rdf = 1 + 1 / r**2 * (np.exp(-(a * r + b)) * np.sin(c * r + d) \\\n",
+    "                                + np.exp(-(g * r + h)) * np.cos(k * r + l))\n",
+    "theo_rdf[np.nonzero(r <= theo_rdf_cutoff)] = \\\n",
+    "                            s * np.exp(-(m * r + n)**4)[np.nonzero(r <= theo_rdf_cutoff)]\n",
+    "\n",
+    "plt.plot(r, theo_rdf, '-', color=\"blue\", linewidth=2, alpha=1, ls='--', label=r'theoretical')\n",
+    "\n",
+    "plt.xlabel('$r [\\sigma]$', fontsize=20)\n",
+    "plt.ylabel('$g(r)$', fontsize=20)\n",
+    "plt.gca().tick_params(axis='both', labelsize=16)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Futher Exercises"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Binary Lennard-Jones Liquid\n",
+    "\n",
+    "A two-component Lennard-Jones liquid can be simulated by placing particles of two types (0 and 1) into the system. Depending on the Lennard-Jones parameters, the two components either mix or separate.\n",
+    "\n",
+    "1. Modify the code such that half of the particles are of <tt>type=1</tt>. Type 0 is implied for the remaining particles.\n",
+    "2. Specify Lennard-Jones interactions between type 0 particles with other type 0 particles, type 1 particles with other type 1 particles, and type 0 particles with type 1 particles (set parameters for <tt>system.non_bonded_inter[i,j].lennard_jones</tt> where <tt>{i,j}</tt> can be <tt>{0,0}</tt>, <tt>{1,1}</tt>, and <tt>{0,1}</tt>. Use the same Lennard-Jones parameters for interactions within a component, but use a different <tt>lj_cut_mixed</tt> parameter for the cutoff of the Lennard-Jones interaction between particles of type 0 and particles of type 1. Set this parameter to $2^{\\frac16}\\sigma$ to get de-mixing or to $2.5\\sigma$ to get mixing between the two components.\n",
```

Comment:
```suggestion
    "2. Specify Lennard-Jones interactions between type 0 particles with other type 0 particles, type 1 particles with other type 1 particles, and type 0 particles with type 1 particles (set parameters for <tt>system.non_bonded_inter[i,j].lennard_jones</tt> where <tt>{i,j}</tt> can be <tt>{0,0}</tt>, <tt>{1,1}</tt>, and <tt>{0,1}</tt>. Use the same Lennard-Jones parameters for interactions within a component, but use a different <tt>lj_cut_mixed</tt> parameter for the cutoff of the Lennard-Jones interaction between particles of type 0 and particles of type 1. Set this parameter to $2^{\\frac{1}{6}}\\sigma$ to get de-mixing or to $2.5\\sigma$ to get mixing between the two components.\n",
```

Reply:
no color highlighting because line is too long: the frac16 at the end was missing braces